README.md

```diff
@@ -31,28 +29,136 @@ If you need BPMN examples, you can use resources from
 - [bpmn-visualization BPMN test diagrams](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/bpmn-files/README.md)
 
 
-## Examples
+## 🌏 Browser Support
 
-Want to know more about `bpmn-visualization` usage and extensibility? Have a look at the
-[__:fast_forward: live examples site__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/index.html).
+| <img src="https://user-images.githubusercontent.com/1215767/34348387-a2e64588-ea4d-11e7-8267-a43365103afe.png" alt="Chrome" width="16px" height="16px" /> Chrome | <img src="https://user-images.githubusercontent.com/1215767/34348383-9e7ed492-ea4d-11e7-910c-03b39d52f496.png" alt="Firefox" width="16px" height="16px" /> Firefox | <img src="https://user-images.githubusercontent.com/1215767/34348394-a981f892-ea4d-11e7-9156-d128d58386b9.png" alt="Safari" width="16px" height="16px" /> Safari | <img src="https://user-images.githubusercontent.com/1215767/34348380-93e77ae8-ea4d-11e7-8696-9a989ddbbbf5.png" alt="Edge" width="16px" height="16px" /> Edge | <img src="https://user-images.githubusercontent.com/1215767/34348590-250b3ca2-ea4f-11e7-9efb-da953359321f.png" alt="IE" width="16px" height="16px" /> Internet Explorer |
+| :---------: | :---------: | :---------: | :---------: | :---------: |
+| Yes | Yes | Yes | Yes | No |
 
-For more technical details and how-to, go to the [bpmn-visualization-examples](https://github.com/process-analytics/bpmn-visualization-examples/)
-repository.
 
+## 🎨 Features
+
+`bpmn-visualization` is in early development stages and is subject to changes prior to the `1.0.0` release.\
+\
+Implemented features:
+- Rendering of the most common BPMN elements.  
+Notice that there is currently no plan to support `Conversation` and `Choreography`.
+
+Current implementation:
+- The [packaging](https://github.com/process-analytics/bpmn-visualization-js/milestone/18) of the library.
+
+Future features:
+- BPMN extensions
+- The library extension points
+- Display options for execution data with interactive capacities.
 
-# Roadmap
 
-`bpmn-visualization` is in early development stages and is subject to changes prior to the `1.0.0` release.
+## 📋 Requirements
 
-For now, we render the most common BPMN elements. Notice that there is currently no plan to support `Conversation` and `Choreography`.
+- `Node.js`: 12.16.x and 14.11.x (may work with other versions but without any guarantee)
+- `Supported OS`: Windows/Linux/MacOs (see the Github Build workflow for more details)
 
-We are currently focusing on the [packaging](https://github.com/process-analytics/bpmn-visualization-js/milestone/18) of the library.
 
-Then, we will work on BPMN extensions, library extension points, display options for execution data with interactive
-capacities.
+## ♻️ Usage
+For now, we don't put the released library on npm. So you need to clone the repository in local, get the last tag, and build it.
```

Comment:
:warning: too fresh to be documented that way, why not waiting for the final packaging and letting people go to the examples repository?
resources below are only available in the examples repository, so at worst, point to the right repository

Reply:
If we want some contributors for HacktoberFest, I think we need this part, event if we change it after the implementation of the packaging.

Reply:
I understand the need of something clearer, but for now, using the lib is complicated and we have to work to make it simpler, for instance with the npm bundles and #478
The examples should be a good starting point and eventually, details should be put there

Reply:
I had a big warning to know that we are working on the packaging